microsoft-teams/1.0/modules/ROOT/pages/microsoft-teams-connector-examples.adoc

```diff
@@ -0,0 +1,427 @@
+
+. Drag the *Create team* operation onto the canvas, to the right of the *Transform Message* component.
+. In the Create team configuration, click the  *Connector configuration* dropdown and select *Microsoft-Teams-Client-Config*.
+. Select `Microsoft_Teams_Client_Credentials_Config` as the global element type and click *OK*.
+. Configure the following fields in the Create team properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |`Create team`
+|Connector Configuration |`OAuth_Client_Credentials_Config`
+|Message |`payload`
+|===
+
+=== Add the Second Transform Message Component
+
+This *Transform Message* component converts the output of the *Create team* operation to JSON format.
+
+. In the Mule Palette view, search for *transform message*:
+. Drag the *Transform Message* component onto the canvas, to the right of the *Listener* component.
+. Click the *Transform Message* component and set the output to `application/json`:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+----
+
+[[retrieve-team]]
+== Retrieve the Team Properties and Relationships
+
+Create a second flow to retrieve the properties and relationships for the new team. Use the *Get Team* operation in this flow:
+
+.Use a flow like this one to retrieve the new team's properties and relationships.
+image::ms-teams-get-team.png[Retrieve the team Flow]
+
+[[retrieve-members]]
+== Retrieve the Team Members
+
+Create a third flow to retrieve information about the members of the new team. Use the *List team members* operation in this flow:
+
+.Use a flow like this one to retrieve information about team members.
+image::ms-teams-get-members.png[Retrieve the team members flow]
+
+[[retrieve-channel-list]]
+== Retrieve the Team Channels
+
+Create a fourth flow to retrieve a information about the channels used by the team. Use the *List channels* operation in this flow:
+
+.Use a flow like this one to retrieve the channels used by the new team.
+image::ms-teams-list-channels.png[Retrieve the team channels flow]
+
+[[onboard-new-hire]]
+== Onboard a New Hire to the Team
+
+Create a fifth flow to onboard a new hire to the team.
+Use the following operations in this flow:
+
+* *Add team member* to add a new member to the team
+* *Create channel* to create a new channel
+* *Add channel member* to add the new member to the newly-created channel
+* *Create message* to create the welcome message
+
+.Use a flow like this one to onboard a new user.
+image::ms-teams-onboarding.png[Onboarding a new hire flow]
+
+[[delete-channel]]
+== Delete a Team Channel
+
+Create a sixth flow to delete a channel. Use the *Delete channel* operation in this flow.
+
+.Use a flow like this one to delete a channel.
+image::ms-teams-delete-channel.png[Delete a channel]
+
+[[remove-team-member]]
+== Remove a Team Member
+
+Create a seventh flow to remove a member from a team. Use the *Remove team member* operation in this flow.
```

Comment:
Does this numbering (th) of flows serve a purpose? Will the audience need to count?